### ISSUE TITLE
Suggestion: Config option to sort alphabetically

### ISSUE THREAD

**Issue by AustP** [tocjent/php-integrator-symbol-viewer#26]
I would like to see a config option that sorts the properties and methods of a class alphabetically.

**Comment by Arcesilas** [comment]
Such sorting should not change the symbols types order, though.
- Static properties together
- Static methods together
- etc.

**Comment by AustP** [comment]
@Arcesilas Yes I agree with that. Keep the grouping, but sort the items in the groups alphabetically.

**Comment by tocjent** [comment]
Would be present in the next release as a config option(coming mid-April). Thank you guys for your help.

**Comment by nahuelhds** [comment]
+1
